sdk/resources/arm-resources/README.md

```diff
@@ -1,109 +1,91 @@
-## Azure ResourceManagementClient SDK for JavaScript
+# Azure ResourceManagement client library for JavaScript
 
-This package contains an isomorphic SDK (runs both in Node.js and in browsers) for ResourceManagementClient.
+This package contains an isomorphic SDK (runs both in Node.js and in browsers) for Azure ResourceManagement client.
+
+Provides operations for working with resources and resource groups.
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/resources/arm-resources) |
```

Comment:
@sarangan12 we are moving to do this in Track 2 readme so please implement my suggestion in code gen:

```suggestion
Key links
[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/resources/arm-resources) |
```